index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="sideNav">
+        <h1>JS documentation</h1>
+        <ul>
+          <li><a href="#intro">Introduction</a></li>
+          <li><a href="#"></a></li>
```

Comment:
Kindly fix this, as it is empty. Either remove it or add a proper `<li>` name for best practices.